pkg/health/health.go

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2022 IBM, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package health
+
+import (
+	"github.com/heptiolabs/healthcheck"
+	"github.com/netobserv/flowlogs2metrics/pkg/config"
+	log "github.com/sirupsen/logrus"
+	"net"
+	"net/http"
+	"time"
+)
+
+const defaultServerHost = "0.0.0.0"
+
+type Server struct {
+	handler healthcheck.Handler
+	address string
+}
+
+func (hs *Server) Serve() {
+	for {
+		err := http.ListenAndServe(hs.address, hs.handler)
+		log.Errorf("http.ListenAndServe error %v", err)
+		time.Sleep(60 * time.Second)
```

Comment:
@KalmanMeth no ... same as in `startPrometheusInterface` 

@KalmanMeth --> Adding exit to those HTTP servers is a complicated task in golang prior to 1.18+ and I do not see a reason (at this stage) to add this complexity to the code. If you really want to do that, we can but I would like to avoid if possilbe.